site/gatsby-site/src/components/discover/Controls.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import REFINEMENT_LISTS from './REFINEMENT_LISTS';
+import styled from 'styled-components';
+import Stats from './Stats';
+import ClearFilters from './ClearFilters';
+import DisplayModeSwitch from './DisplayModeSwitch';
+import Filters from './Filters';
+import { Row, Col } from 'react-bootstrap';
+
+const FlexGap = styled(Col)`
+  margin: auto;
+`;
+
+const ExpandFilters = styled.button`
+  user-select: none;
+  cursor: pointer;
+  color: inherit;
+  background: none;
+  border: none;
+`;
+
+const Controls = ({ query }) => {
+  const [expandFilters, setExpandFilters] = useState(false);
+
+  useEffect(() => {
+    let filter = false;
```

Comment:
No need to change this but `some` can save you some lines in the future, and the `if` is redundant.
```
const filter = REFINEMENT_LISTS.some(r => query[r.attribute]);
setExpandFilters(filter);
```